portfolio/src/main/webapp/scripts/cookies.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Create a new cookie based on name, value and days before expiring.
+ */
+function setCookie(name, value, expirationDays) {
+  let date = new Date();
+  date.setTime(date.getTime+(expirationDays*24*60*60*1000));
```

Comment:
[Nit] It's nice to have constants when doing long multiplications like this.  I might do the following
HOURS_PER_DAY = 24
MINUTES_PER_HOUR = 60
SECONDS_PER_MINUTE = 60
SECONDS_PER_DAY = SECONDS_PER_MINUTE * SECONDS_PER_DAY * HOURS_PER_DAY
MILLISECONDS_PER_SECOND = 1000

and then down here use
expirationDays*MILLISECONDS_PER_SECOND*SECONDS_PER_DAY